init.sls

```diff
@@ -67,7 +67,9 @@ ssh-keyscan {{ pillar['duplicity']['ssh']['known_hosts'] }} >> /etc/ssh/ssh_know
     - defaults:
       remove_all_but_n_full: {{ pillar['duplicity']['remove_all_but_n_full']|default(5) }}
       full_if_older_than: {{ pillar['duplicity']['full_if_older_than']|default('30D') }}
-      include_dirs: {{ pillar['duplicity']['include_dirs']|default(['/etc', '/root']) }}
+      data_dir: {{ salt['pillar.get']('duplicity:data_dir', '/') }}
+      include_dirs: {{ salt['pillar.get']('duplicity:include_dirs', []) }}
```

Comment:
I think this is a valid choice (because of the configurable root dir), but it might break existing setups. 

So this should be documented in the CHANGELOG and requires a major version bump.